src/test/unit/controls/Gantt/Gantt-spec.js

```diff
@@ -2284,6 +2304,27 @@ describe("Gantt", () => {
                 done();
             }, TRANSITION_DELAY);
         });
+        it("Removes multiple content successfully", (done) => {
+            gantt.destroy();
+            gantt = new Gantt(getAxes(axisJSON));
+            const multipleContents = [primaryContent, secondaryContent];
+            gantt.loadContent(multipleContents);
+            expect(gantt.tracks.length).toBe(2);
+            gantt.unloadContent(multipleContents);
+            delay(() => {
+                expect(gantt.tracks).toEqual([]);
+                expect(gantt.trackConfig).toEqual([]);
+                expect(gantt.tracks.length).toBe(0);
+                expect(gantt.trackConfig.length).toBe(0);
+                expect(gantt.config.height).toBe(0);
+                expect(gantt.config.axis.y.domain.lowerLimit).toBe(0);
+                expect(gantt.config.axis.y.domain.upperLimit).toBe(1);
+                expect(gantt.config.canvasHeight).toBe(
+                    BASE_CANVAS_HEIGHT_PADDING
+                );
+                done();
+            }, TRANSITION_DELAY);
+        });
```

Comment:
Can you update the documentation as well? We would want to advertise that the data can be loaded as array now. Refer [pie.md](https://github.com/cerner/carbon-graphs/blob/master/docs/controls/Pie.md#data) `Usage`, `JSON Properties` and `Structure`

Reply:
done